### ISSUE TITLE
Service: spawn parameters display and edit

### ISSUE THREAD

**Issue by dee-ynput** [ynput/ayon-frontend#956]
### Problems
There is no way to check what options were used to spawn a service. For example the storages. 

### Proposal
- Have frontend display the values used for service start.
- Have an option to edit the parameters and restart the service with the new values.

**Comment by Innders** [comment]
The columns contain most of the service information.

You could have a context menu to update the service, this would open the configuration dialog for the service so you can it's current configuration and change it if needed.

![Image](https://github.com/user-attachments/assets/01eec3e2-6119-424c-b693-cd743b2c3970)

**Comment by dee-ynput** [comment]
I believe the primary concern was about the storage option. Am I right @m-u-r-p-h-y ?

**Comment by m-u-r-p-h-y** [comment]
> I believe the primary concern was about the storage option. Am I right [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/352795?s=64&amp;u=d2a3cc7174303dbf30b503c29e2b990f19f61fbe&amp;v=4">@m-u-r-p-h-y](https://github.com/m-u-r-p-h-y?rgh-link-date=2024-12-11T08%3A39%3A44.000Z) ?

Yes. There is no way to see the storage info and if you are setting up the service the syntax for the storage can be quite tricky. So you will end up in the loop of creating a new service - trying if it works - deleting the service - spawning a new one with corrected options and all over again.
